subworkflows/local/pair_variant_calling.nf

```diff
@@ -188,14 +193,32 @@ workflow PAIR_VARIANT_CALLING {
         ch_versions = ch_versions.mix(GATK_TUMOR_NORMAL_SOMATIC_VARIANT_CALLING.out.versions)
     }
 
-    // if (tools.contains('tiddit')) {
-    // }
+    //TIDDIT
+    if (tools.contains('tiddit')){
+        cram_normal = cram_pair.map{meta, normal_cram, normal_crai, tumor_cram, tumor_crai ->
+            [meta, normal_cram, normal_crai]
+        }
+        cram_tumor = cram_pair.map{meta, normal_cram, normal_crai, tumor_cram, tumor_crai ->
+            [meta, tumor_cram, tumor_crai]
+        }
+        RUN_TIDDIT_NORMAL(cram_normal, fasta, bwa)
+        RUN_TIDDIT_TUMOR(cram_tumor, fasta, bwa)
+        svdb_input = RUN_TIDDIT_NORMAL.out.tiddit_vcf.join(RUN_TIDDIT_TUMOR.out.tiddit_vcf)
+        svdb_input_correct = svdb_input.map{ meta, vcf_normal, vcf_tumor -> [meta, [vcf_normal, vcf_tumor]]}
+        SVDB_MERGE(svdb_input_correct, false)
```

Comment:
```suggestion
        SVDB_MERGE(RUN_TIDDIT_NORMAL.out.tiddit_vcf.join(RUN_TIDDIT_TUMOR.out.tiddit_vcf).map{ 
                            meta, vcf_normal, vcf_tumor -> 
                            [meta, [vcf_normal, vcf_tumor]]
                            }, false)
```

This might not be indented properly

Reply:
How many indents are actually necessary according to convention? Should `meta,...` align to the map command or is 1 indent against `SVDB_MERGE` enough?

Reply:
excellent question. Since we don't have a linter it is only self-enforced so obviously sometimes there are slight variations. the goal should be that it is well readable, for that I usually try to work with empty lines between processes and align channel operator on `.`. Some leniency on the alignment part has to be allowed sometimes, because the editorconfig is complaining if it is not multiple of 4. 

so  in the above case, I would align `.join` and `.map`, keep `meta, vcf_normal, vcf_tumor -> ` in the same line as `map` and the align the curly bracket with map again